samples/java_springboot/16.proactive-messages/src/main/java/com/microsoft/bot/sample/proactive/ProactiveBot.java

```diff
@@ -32,7 +32,8 @@
     @Value("${server.port:3978}")
     private int port;
 
-    private static final String WELCOMEMESSAGE =
+    // Message to send to users when the bot receives a Conversation Update event
+    private final String WELCOMEMESSAGE =
```

Comment:
```suggestion
    private final String welcomeMessage =
```

As this is not static